### PR TITLE
Added trained model

### PR DESCRIPTION
**Pull request by Azamatas**

### CODE REVIEW

File: local_model/timer.py
Comment (jesyspa) [inline local_model/timer.py:1]: I don't quite get the filename, is timing the main responsibility of this module?

File: local_model/timer.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        end = time.time()
+    return (end - start) / n
+
+def generate_answer(model, inputs):
+    with torch.no_grad():
+        output = model.generate(**inputs, max_new_tokens=50, do_sample=False)
+    return tokenizer.decode(output[0], skip_special_tokens=True)
+
+print("Base model avg time:", benchmark(base_model, inputs))
+print("Finetuned model avg time:", benchmark(finetuned_model, inputs))
```
Comment (jesyspa) [inline local_model/timer.py]: Is comparing this meaningful?  Do we expect finetuning to impact performance?

File: local_model/timer.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+print("Base output:")
+print(generate_answer(base_model, inputs))
```
Comment (jesyspa) [inline local_model/timer.py]: Similarly here - being able to get an answer is nice, but it doesn't seem likely to be a meaningful way to compare the two (at least, not for single answers).

**Review by jesyspa — CHANGES_REQUESTED** [review]
I'd like to have a clearer understanding of the goal here.  Your PR doesn't contain a model (and probably it isn't a good idea to put models on GitHub, they're quite big), so the description isn't accurate.  What problem is this change addressing?

File: local_model/timer.py
Comment (Azamatas) [reply local_model/timer.py:1]: It was originally to compare times of execution of trained and untrained models, but i added an output of their generated text afterwards

File: local_model/timer.py (hunk 1 above)
Comment (Azamatas) [reply local_model/timer.py]: I was able to deduct that unfinetuned model wasnt generating my custom code. This compare, however, is not meaningful as finetuning does not change performance

File: local_model/timer.py (hunk 2 above)
Comment (Azamatas) [reply local_model/timer.py]: I think I can add comparison by "loss" of the models

File: local_model/timer.py
Comment (Azamatas) [reply local_model/timer.py:1]: The training outcomes are two checkpoints folders, that contain the model’s weight shards

Comment (jesyspa) [reply local_model/timer.py:1]: These folders are not in the git repository.  How big are they?  Are you sure it's a good idea to include them?

File: local_model/timer.py (hunk 2 above)
Comment (jesyspa) [reply local_model/timer.py]: Okay, could you implement that, or would it be too complicated?

File: local_model/timer.py
Comment (Azamatas) [reply local_model/timer.py:1]: They are about 200MB together,however, i am not sure that its appropriate for them tobe included in git

File: local_model/timer.py (hunk 2 above)
Comment (Azamatas) [reply local_model/timer.py]: No, its trivial. I can do it

File: local_model/timer.py
Comment (jesyspa) [reply local_model/timer.py:1]: I would recommend against putting files of that size in a git repo.

**Review by jesyspa — APPROVED** [review]
